docs/rules/lines-around-comment.md

```diff
@@ -1,35 +1,11 @@
-# Enforce empty lines around comments (lines-around-comment)
+# require empty lines around comments (lines-around-comment)
 
 Many style guides require empty lines before or after comments. The primary goal
 of these rules is to make the comments easier to read and improve readability of the code.
 
 ## Rule Details
 
-This rule allows you to specify whether an empty line should be found
-before or after a comment. It can be enabled separately for both block (`/*`)
-and line (`//`) comments, and does not apply to comments that appear on the same
-line as code.
-
-
-By default an empty line is required above a block comment,
-such as in the following example:
-
-```js
-var x = 0;
-
-/**
- * The vertical position.
- */
-var y = 10;
-```
-
-The following would *not* pass the rule:
-
-```js
-var x = 0;
-/* the vertical position */
-var y = 10;
-```
+This rule requires empty lines before and/or after comments. It can be enabled separately for both block (`/*`) and line (`//`) comments. This rule does not apply to comments that appear on the same line as code and does not require empty lines at the beginning or end of a file.
 
 ### Inline comments
```

Comment:
The improved wording under h2 Rule Details makes the section under h3 Inline comments obsolete, I think.